src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -190,19 +240,21 @@ export class SearchMenu extends React.Component {
                 className="usagov-search-autocomplete  vads-u-flex--4 vads-u-margin-left--1 vads-u-margin-right--0p5 vads-u-margin-y--1  vads-u-width--full"
                 name="query"
                 aria-controls={isOpen ? 'suggestions-list' : undefined}
+                onFocus={getSuggestions}
+                onKeyUp={handleKeyUp}
                 {...getInputProps({
                   type: 'text',
-                  onChange: this.handleInputChange,
+                  onChange: handleInputChange,
                   'aria-labelledby': 'site-search-label',
                   id: 'query',
-                  onKeyUp: this.handleKeyUp,
                 })}
               />
               <button
                 type="submit"
                 disabled={!validUserInput}
```

Comment:
Hmm, I don't think this will work for the case where a user presses enter instead of clicking on the button